src/libs/caching.js

```diff
@@ -1,5 +1,7 @@
 import { debug, error } from "./logging";
-import { storage } from "webextension-polyfill";
+import Browser, { storage } from "webextension-polyfill";
```

Comment:
```suggestion
import { runtime, storage } from "webextension-polyfill";
```

I'm not sure whether importing the whole `Browser` object messes with the tree shaking. I think it'd be better to just import the runtime part of it and adjust the code in the rest of the file accordingly.